modules/build-images/dockerfile/common/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
What would be the reason for duplicating contents of docker-common in this repository?